### DISCUSSION TITLE
Pushover Support for Music Sync.

### DISCUSSION THREAD

**Discussion by s-j-daniels** [marcone/teslausb#787] [General]
I use Pushover for DashCam and Sentry Backups to my NAS. Everything works perfectly. I would like to know if I can add Pushover support for the Music Archive as well.

As it functions now I get no notifications on Music synchronizations, I have to keep checking the archive log files for a completion.

**Comment by marcone** [comment]
What does it say in the archive loop logs? It is already sending a notification for music sync, independent of which notification method you use. It should look something like this:
```
Copied 1 music file(s), deleted 0, skipped 254 previously-copied files, and encountered 0 errors
```
Note that it only sends notification for music if it actually did anything. If there was nothing to do for music, you won't get a notification.

**Reply by s-j-daniels** [reply]
I just received a message, it's actually why I came back to this post. I'm not sure why I have never received any before, but 100% honesty I re-flashed my SSD this afternoon and started from scratch. I have issues with the MCU2 shutting down the USB ports so I put a blank USB in the extra port an hour ago. So my assumption is the Pi was shutting down before completion and the unit was powered long enough to update saved video and send a message.

Sun 15 Oct 18:37:28 EDT 2023: Copied 330 music file(s), deleted 2, skipped 692 previously-copied files, and encountered 0 errors.

![IMG_1024](https://github.com/marcone/teslausb/assets/19993862/a03e11dc-1540-46de-9ed1-f565fe42e9b4)
